docs/concepts/cache.md

```diff
@@ -104,4 +104,4 @@ uv determines the cache directory according to, in order:
 
 It is important for performance for the cache directory to be located on the same file system as the
 Python environment uv is operating on. Otherwise, uv will not be able to link files from the cache
-into the environment and will instead need to fallback to slow copy operations.
+into the environment and you will instead need to use copy mode with slow operations.
```

Comment:
Why this change? We're talking about uv here, e.g., "uv will instead need to fallback to slow copy operations"

Reply:
by reading `fallback`, initially I thought it will automatically do it for us, because imo it means: if some progress failed do X instead.

Reply:
It does / should do it automatically for you?

Reply:
Then I think Warning message could be improved to mention which mode it fallbacking, in #6397 it says:
```
         If the cache and target directories are on different filesystems, hardlinking may not be supported.
         If this is intentional, set `export UV_LINK_MODE=copy` or use `--link-mode=copy` to suppress this warning.
```
Instead:
```
         Cache and target directories are on different filesystems, hardlinking not supported, switched to copy mode.
         If this is intentional, set `export UV_LINK_MODE=copy` or use `--link-mode=copy` to suppress this warning.
```

Reply:
Probably worth discussion elsewhere — we have too many issues to go through right now to discuss unrelated changes out of context.